docs: How-to guides review

sdiazlor opened the pull request:
# Description
<!-- Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change. -->

Closes #5332 

**Type of change**
<!--  Please delete options that are not relevant. Remember to title the PR according to the type of change  -->

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- Refactor (change restructuring the codebase without changing functionality)
- Improvement (change adding some improvement to an existing functionality)
- Documentation update

**How Has This Been Tested**
<!--  Please add some reference about how your feature has been tested.  -->

**Checklist**
<!--  Please go over the list and make sure you've taken everything into account -->

- I added relevant documentation
- I followed the style guidelines of this project
- I did a self-review of my code
- I made corresponding changes to the documentation
- I confirm My changes generate no new warnings
- I have added tests that prove my fix is effective or that my feature works
- I have added relevant notes to the CHANGELOG.md file (See https://keepachangelog.com/)

sdiazlor committed:
mkdocs uncomment script

sdiazlor committed:
delete old HF set up

sdiazlor committed:
dataset guide

sdiazlor committed:
distribution guide

sdiazlor committed:
query guide

dvsrepo committed:
Add info about workspaces

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks
for more information, see https://pre-commit.ci

sdiazlor committed:
import_export guide

sdiazlor committed:
record guide

sdiazlor committed:
use_markdown guide

sdiazlor committed:
workspace guide

sdiazlor committed:
user guide

frascuchon commented on argilla/docs/how_to_guides/record.md:
```diff
@@ -468,23 +468,23 @@ dataset.records.log(records=updated_data)
-        record.vectors["new_vector"] = [ 0, 1, 2, 3, 4, 5 ]
-        record.vectors["v"] = [ 0.1, 0.2, 0.3 ]
+        record.vectors["my_vector"] = [ 0, 1, 2, 3, 4, 5 ]
+        record.metadata["my_new_vector"] = [ 0, 1, 2, 3, 4, 5 ]
```
```suggestion
        record.vectors["my_new_vector"] = [ 0, 1, 2, 3, 4, 5 ]
```

sdiazlor committed:
Update argilla/docs/how_to_guides/record.md

Co-authored-by: Paco Aranda <frascuchon@gmail.com>

sdiazlor committed:
Merge branch 'docs/2.0.0' into docs/review-code-snippets

frascuchon committed:
[DOCS] Add howto work with setting when listing datasets (#5339)

# Description
<!-- Please include a summary of the changes and the related issue.
Please also include relevant motivation and context. List any
dependencies that are required for this change. -->

This PR adds a section of how to work with settings when listing
datasets.

**Type of change**
<!-- Please delete options that are not relevant. Remember to title the
PR according to the type of change -->

- Documentation update

**How Has This Been Tested**
<!-- Please add some reference about how your feature has been tested.
-->

**Checklist**
<!-- Please go over the list and make sure you've taken everything into
account -->

- I added relevant documentation
- I followed the style guidelines of this project
- I did a self-review of my code
- I made corresponding changes to the documentation
- I confirm My changes generate no new warnings
- I have added tests that prove my fix is effective or that my feature
works
- I have added relevant notes to the CHANGELOG.md file (See
https://keepachangelog.com/)

sdiazlor committed:
Merge branch 'docs/2.0.0' into docs/review-code-snippets

sdiazlor committed:
update initial users/workspaces

sdiazlor committed:
Update link
